pyquil/noise.py

```diff
@@ -30,7 +30,7 @@
 
 if TYPE_CHECKING:
     from pyquil.quil import Program
-    from pyquil.api import QuantumComputer
+    from pyquil.api import QuantumComputer as PyquilApiQuantumComputer
```

Comment:
Otherwise you get this: https://github.com/rigetti/pyquil/actions/runs/3154031621/jobs/5131175895

```
/home/runner/work/pyquil/pyquil/pyquil/noise.py:
docstring of pyquil.noise.estimate_assignment_probs:1:

more than one target found for cross-reference 'QuantumComputer':
- pyquil.api.QuantumComputer
- pyquil.api._quantum_computer.QuantumComputer
- pyquil.compatibility.v2.api.QuantumComputer
- pyquil.compatibility.v2.api._quantum_computer.QuantumComputer
```